package.json

```diff
@@ -17,7 +17,8 @@
 		"changeset:version": "changeset version && pnpm -r generate:version && git add --all",
 		"changeset:release": "changeset publish",
 		"start": "cd sites/kit.svelte.dev && npm run dev",
-		"postinstall": "pnpm -r generate:types"
+		"build": "pnpm --filter @sveltejs/* -r build",
```

Comment:
what is this building? almost all the packages are build-less

Reply:
> Adds a root build script which should only run build on things like adapters that have various scripts that need to be run for them to work. I also added it to postinstall so you don't need to think about running it before trying to dev anything